catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/QueryFilterTransformerProvider.java

```diff
@@ -47,32 +52,64 @@ public synchronized void bind(ServiceReference<QueryFilterTransformer> reference
 
     for (QName namespace : namespaces) {
       queryFilterTransformerMap.put(namespace, transformer);
+      List<String> typeNames = getTypeNames(reference);
+      if (CollectionUtils.isNotEmpty(typeNames)) {
```

Comment:
This `if` statement is unnecessary; `getTypeNames` is not `@Nullable` and never returns `null` and if the list is empty, the `for` loop inside will not iterate.

Reply:
removed if statement